packages/sanity/src/core/releases/tool/BundleDetail.tsx

```diff
@@ -16,7 +16,7 @@ export const BundleDetail = () => {
   const [activeScreen, setActiveScreen] = useState<Screen>('overview')
   const {bundleId}: ReleasesRouterState = router.state
   const parsedBundleId = decodeURIComponent(bundleId || '')
-  const {data, loading} = useBundlesStore()
+  const {data, loading} = useBundles()
   const bundleDocuments = [] // TODO: fetch docs with bundle version
```

Comment:
unrelated to this PR / outside of scope: but I have a groq query with this in the util in the dummyGetters (I kept it there since I assume we want to do something different in the future when fetching them, but in the meantime it does what it says on the tin)

Reply:
Spoke with Pedro, and it was my own confusion! resolving the conversation!